src/main/java/yapper/app/Ui.java

```diff
@@ -0,0 +1,67 @@
+package yapper.app;
+
+/**
+ * Provides utility methods for interacting with the user via the command line.
+ * Includes methods for displaying messages and errors.
+ */
+public class Ui {
+
+    private static final String DIVIDER = "-------------------------------------------------";
+    private static final String NAME = "Yapper";
+
+    /**
+     * Constructs a Ui object.
+     */
+    public Ui() {
+    }
+
+    /**
+     * Displays the introduction message and prompts the user.
+     */
+    public static String intro() {
```

Comment:
introduceMyself() might be better here as it is a method.

Reply:
Yes, it should be more descriptive, thank you for the suggestion.